stock_chart/endpoint.erb

```diff
@@ -0,0 +1,31 @@
+# symbols for qandl are often specified by
```

Comment:
This comment is repeated in the widget, seems like it would be better to have it in only one place. I suggest removing it here and keeping it in the widget where you're actually going to supply the param.